_wallets/electrum.md

```diff
@@ -41,6 +43,7 @@ platform:
         link: "https://play.google.com/store/apps/details?id=org.electrum.electrum"
         source: "https://github.com/spesmilo/electrum"
         screenshot: "electrumandroid.png?1528322191"
+        features: "bech32 legacy_addresses segwit"
```

Comment:
@alexcherman 

Features may differ depending on environment - added values to reference underneath desktop and mobile os type (if a wallet is available in multiple environments).